src/components/new-safe/CreateSafe/index.tsx

```diff
@@ -24,37 +26,71 @@ export type NewSafeFormData = {
   mobileOwners: NamedAddress[]
 }
 
-export const CreateSafeSteps: TxStepperProps<NewSafeFormData>['steps'] = [
-  {
-    title: 'Connect wallet',
-    subtitle: 'In order to create a Safe you need to connect a wallet',
-    render: (data, onSubmit, onBack, setStep) => (
-      <CreateSafeStep0 data={data} onSubmit={onSubmit} onBack={onBack} setStep={setStep} />
-    ),
+const staticHints: Record<
+  number,
+  { title: string; variant: AlertColor; steps: { title: string; text: string | ReactElement }[] }
+> = {
+  1: {
+    title: 'Safe Creation',
+    variant: 'info',
+    steps: [
+      {
+        title: 'Network fee',
+        text: 'Deploying your Safe requires the payment of the associated network fee with your connected wallet. An estmation will be provided in the last step.',
+      },
+    ],
   },
-  {
-    title: 'Select network and name Safe',
-    subtitle: 'Select the network on which to create your Safe',
-    render: (data, onSubmit, onBack, setStep) => (
-      <CreateSafeStep1 data={data} onSubmit={onSubmit} onBack={onBack} setStep={setStep} />
-    ),
+  2: {
+    title: 'Safe Creation',
+    variant: 'info',
+    steps: [
+      {
+        title: 'Flat hierarchy',
+        text: 'Every owner has the same rights within the Safe and can propose, sign and execute transactions.',
+      },
+      {
+        title: 'Managing Owners',
+        text: 'You can always change the amount of owners and required confirmations in your Safe at a later stage after creation.',
+      },
+      {
+        title: 'Safe Setup',
+        text: (
+          <>
+            Not sure how many owners and confirmations you need for your Safe?{' '}
+            <Link
+              sx={{ textDecoration: 'underline' }}
```

Comment:
Should we override the global link styles? cc @liliiaorlenko

Reply:
We don't have a mockup for the link. Mostly because it was only suggested in this PR. I suggest it should not block this one and rather be addressed after a final design review on the Create flow

Reply:
I meant that our theme includes link styling and underlining it here does not conform with it. We can revisit this though.